app/components/plans/newplan.js

```diff
@@ -0,0 +1,142 @@
+import React from 'react';
+import {connect} from 'react-redux';
+import {DropDownMenu, MenuItem, Divider, FloatingActionButton, GridList,GridTile, TextField, SelectField} from 'material-ui';
+import ContentAdd from 'material-ui/svg-icons/content/add';
+
+
+//======FAKE DATA ============
+const fakeExerciseArray = [{
+  "id" : 1,
+  "title" : "Horizontal Abduction",
+  "description" : " Start by lying on your stomach and your arm dangling",
+  "img_URL" : "http://www.stellarhealthcenter.com/Exercises=Stretch/resized/Shoulder-Prone-Horizontal-Abduction.jpg",
+  "vid_url" : ""
+ },
+ {
+   "id" : 2,
+   "title" : "Wall or Table Push Up",
+   "description" : "To do this correctly, raise your arms up in front of you so your arms are even with the ground. Then try and make your arms a little longer by bringing your shoulders forward. This is the +, it is what you want to feel at the end each push up.",
+   "img_URL" : "http://www.stellarhealthcenter.com/Exercises=Stretch/resized/shoulder-wall-push-up-start.jpg",
+   "vid_url" : ""
+  },
+  { "id" : 3,
+    "title" : "Prone External Rotation",
+    "description" : "Start by lying on your stomach and your arm dangling",
+    "img_URL" : "",
+    "vid_url" : ""
+  }
+];
+//============================
+//=======  Component==========
+class newPlan extends React.Component{
+  constructor(props){
+    console.log(props);
+    super(props);
+
+    this.state={
+      patient : {},
```

Comment:
we don't need patient on local state (per our conversation about patients on the store). I suggest a mapStateToProps where you grab the current patient from the list of patients